src/stores/Store.js

```diff
@@ -1,10 +1,21 @@
 // @flow
 
 import { observable } from 'mobx';
-import { Plan } from './models/Plan';
+import { PlanModel } from './models/PlanModel';
 
 export class Store {
 
-  @observable mainPlan: Plan = new Plan();
+  @observable mainPlan = new PlanModel();
+
+  constructor() {
+    this.mainPlan.addYear();
+    this.mainPlan.years[0].addTerm();
+    this.mainPlan.years[0].terms[0].addCourse('Intro to SE', 'SWEN', '261', 3);
+    this.mainPlan.years[0].terms[0].addCourse('Computer Science One', 'CSCI', '141', 4);
+    this.mainPlan.years[0].terms[0].addCourse('Project Based Calculus One', 'MATH', '141', 4);
+    this.mainPlan.addColor('SWEN', 'blue');
+    this.mainPlan.addColor('CSCI', 'green');
+    this.mainPlan.addColor('MATH', 'red');
```

Comment:
We can do this here or in the next PR, but seeing dnd across multiple columns would be nice. Probably a "next PR" problem though.